src/Tempest/Generation/src/HasGeneratorCommand.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Generation;
+
+use Tempest\Console\Console;
+use Tempest\Core\Composer;
+use Tempest\Support\PathHelper;
+use function Tempest\Support\str;
+use Tempest\Validation\Rules\EndsWith;
+use Tempest\Validation\Rules\NotEmpty;
+
+/**
+ * This provide bunch of methods to generate files.
+ * Also methods to manipulate the user generic input like the class name.
+ */
```

Comment:
I love docblocks that describe the purpose of a class. Well done ❤️ 
```suggestion
/**
 * Provides a bunch of methods to generate files and work with common user input.
 */
```

Reply:
Thanks, but thank you for updating them from an external reading POV 🙏